board/apiwrappers.py

```diff
@@ -0,0 +1,115 @@
+from requests import get, post
+
+
+API_IGDB_URL = 'https://api.igdb.com/v4/'
+API_TWITTER_URL = 'https://api.twitter.com/2/'
+
+
+class BaseWrapper:
+    def __init__(self, auth_token):
```

Comment:
Not sure if it makes sense to specifiy auth token here since many apis have different ways of accessing the info.